Makefile

```diff
@@ -482,40 +485,66 @@ proto-check-breaking:
 	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=main
 
 
-TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.15/proto/tendermint
+TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.5/proto/tendermint
 GOGO_PROTO_URL      = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
-COSMOS_SDK_URL      = https://raw.githubusercontent.com/ingenuity-build/cosmos-sdk/v0.45.1-ls-2
-COSMOS_PROTO_URL    = https://raw.githubusercontent.com/cosmos/cosmos-proto/main/proto/cosmos_proto
+CONFIO_URL          = https://raw.githubusercontent.com/confio/ics23/v0.7.0
+SDK_PROTO_URL 		= https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.41.0/proto/cosmos
```

Comment:
Similarly, this should be:

`https://raw.githubusercontent.com/ingenuity-build/cosmos-sdk/v0.45.4-ls.1/proto/cosmos`

Reply:
updated